Add 2023-03-15 agenda and minutes (#511)
solid/specification: meetings/2023-03-15.md

### Participation and Code of Conduct

> * [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement]( )

Author: It apparently changed during the meeting. We mentioned CLA briefly so someone must've cut the text instead of copy. Thanks for catching that.

* SC: Are we aligned on the purpose of the repo? Any objections/comments. There was call to have a repo on shapes. If that is not satisfactory, we can revisit this topic.
* SC: No objections to creation of the shapes repo!

ACTION: SC to create `solid/shapes` repo.

### AuthN and AuthZ Server side clients (apps)
URL: https://github.com/solid/specification/issues/504

* SC: This is a follow-up/checkin on this issue as previous meetings have discussed this topic. I believe eP created this issue, can eP provide an status update? Jackson won't be able to work on it till later this week.
* eP: The issue includes 3 specific design choices which so far received 0 feedback. 
* eP: the issue is currently inactive.
* SC: Suggest we come back to the issue. If anyone in this meeting has processed the issue and has comments, please share your experience/comments.
* SC: Since we don't have feedback, we'll come back to this issue in the future. There is a lot of material to digest. When eP feels better, maybe he can provide a primer on the issue to help people understand where to start. 

> * eP: It links back to another relevant issue about CLI clients: https://github.com/solid/data-interoperability-panel/issues/178

Reviewer: ```suggestion
* eP: It links back to [another relevant issue about CLI clients](https://github.com/solid/data-interoperability-panel/issues/178)
```

> * SC: Specify kind of actors and class of interactions: https://github.com/solid/specification/issues/36 (c. 2019)

Reviewer: ```suggestion
* SC: [Specify kind of actors and class of interactions](https://github.com/solid/specification/issues/36) (circa 2019)
```

> * SC: there has been wok on this topic over the past few years, but eP is helping to move this issue further. Question: will we look at server-to-server interoperability. Most conversations have been on client-server, but server-to-server interaction is increasingly important (as expected).

Reviewer: ```suggestion
* SC: there has been work on this topic over the past few years, but eP is helping to move this issue further. Question: will we look at server-to-server interoperability? Most conversations have been on client-server, but server-to-server interaction is increasingly important (as expected).
```

### Auxiliary Resources with own Access Controls
URL: https://github.com/solid/specification/issues/501

* SC: About time we have a closer look at this for a future milestone?
* SC: This is a deep topic, probably can't get into too deeply today given the time remaining in this meeting. This topic touches upon access controls for profiles and resources (primary, auxiliary). Clarification is needed regarding what (specific) access is allowed, which depends on the type of resource

> * JZ: I recall you can have a ACL on a describedby resource. Is this still part of the discussion?
> * SC: This is a great example of this issue. So far that pattern has been in an implementation (e.g., NSS) but not specified. Along with ACLs having own ACLs. The WAC spec for example does not constrain which resources can have an ACL - so any resource can. We need to clarify in the Solid Protocol as there are cases in which separating the controls on the primary resource and the associated resource. See issue for some examples.

Reviewer: ```suggestion
* JZ: I recall you can have a ACL on a `describedby` resource. Is this still part of the discussion?
* SC: This is a great example of this issue. So far that pattern has been in an implementation (e.g., NSS) but not specified. Along with ACLs having their own ACLs. For example, the WAC spec does not constrain which resources can have an ACL — so any resource can. We need to clarify this in the Solid Protocol, as there are cases in which the controls on the primary resource and the associated resource are separated. See issue for some examples.
```

> * eP: I think we can't avoid paying attention to UX patters that might be required. If we expect end user to manage access policies we need to provide them with something that an average (non-geek) person can manage in responsible way.
> * AD: How complex is this going to be for users - with what Solid provides. Solid has many topics, .. so end user using some component, so is the complexity going to be too much for them or some abstraction that simplifies it for them. Especially using different applications that may not be uniform then it'd be difficult.

Reviewer: ```suggestion
* eP: I think we can't avoid paying attention to UX patterns that might be required. If we expect end users to manage access policies, we need to provide them with something that an average (non-geek) person can manage in a responsible way.
* AD: How complex is this going to be for users — with what Solid provides? Solid has many topics, ... so end user using some component, is the complexity going to be too much for them, or some abstraction that simplifies it for them? Especially using different applications that may not be uniform, it'd be difficult.
```

> * SC: WAC authorization model is designed to simplify (e.g., meant to be basic read/write/control). Users may also think in more of a workflow perspective, for example review of item.
> * eP: I'm happy to share some of the approach we took in the SAI spec. We could have short agenda point next week. SAI: https://solid.github.io/data-interoperability-panel/specification/

Reviewer: ```suggestion
* SC: WAC authorization model is designed to simplify (e.g., meant to be basic read/write/control). Users may also think in more of a workflow perspective, such as review of item.
* eP: I'm happy to share some of the approach we took in [the SAI spec](https://solid.github.io/data-interoperability-panel/specification/). We could have a short agenda point next week.
```
